smaug/python/unique_name_test.py

```diff
@@ -77,5 +77,11 @@ def test_user_supplied_names4(self):
       res = math_ops.add(res, res, name="add")
     self.assertEqual(get_node_names(test_graph), {"add", "add_1", "add_2"})
 
+  def test_user_supplied_names5(self):
```

Comment:
I don't see what this test tests that the previous ones don't?

Reply:
This one tests the case where the name was not in use and `mark_as_used` is set to False.